core/src/test/java/com/rudderstack/core/AnalyticsTest.kt

```diff
@@ -827,6 +857,23 @@ abstract class AnalyticsTest {
         verify(infraPlugin, times(1)).reset()
     }
 
+    private fun generateDataOfSize(msgSize: Int): String {
+        return CharArray(msgSize).apply { fill('a') }.joinToString("")
+    }
+
+    private fun getNumberOfBatches(message: Message?, totalMessages: Int): Int {
```

Comment:
Can you explain this method? Also the method and param names are not self explanatory.

Reply:
I've improved it. Please review it again.